app/scripts/components/common/map/style-generators/raster-timeseries.tsx

```diff
@@ -272,14 +272,36 @@ export function RasterTimeseries(props: RasterTimeseriesProps) {
         LOG && console.log('Payload', payload);
         LOG && console.groupEnd();
         /* eslint-enable no-console */
+        
+        let responseData;
```

Comment:
since the data is not needed outside of the scope, I think we can scope this into each block, (then removing the old code bits will be even easier I think.)

Reply:
responseData is still being referenced here though https://github.com/NASA-IMPACT/veda-ui/blob/80687e78127a3f57a998cd98646d4a6b591ad655/app/scripts/components/common/map/style-generators/raster-timeseries.tsx#L314-L315